src/CommunityToolkit.Maui/Behaviors/AnimationBehavior.shared.cs

```diff
@@ -30,11 +30,17 @@ public partial class AnimationBehavior : EventToCommandBehavior
 	public static readonly BindableProperty AnimateCommandProperty =
 		BindableProperty.CreateReadOnly(nameof(AnimateCommand), typeof(Command<CancellationToken>), typeof(AnimationBehavior), default, BindingMode.OneWayToSource, propertyChanging: OnAnimateCommandChanging, defaultValueCreator: CreateAnimateCommand).BindableProperty;
 
+	/// <summary>
+	/// Backing BindableProperty for the <see cref="AnimateOnTap"/> property.
+	/// </summary>
+	public static readonly BindableProperty AnimateOnTapProperty =
+		BindableProperty.Create(nameof(AnimateOnTap), typeof(bool), typeof(AnimationBehavior), propertyChanged: OnAnimateOnTapPropertyChanged);
```

Comment:
Sorry to bring something back up but I just thought, the original behaviour was to add the gesture recogniser right? I wonder if we should default this property to true which means this won't break existing users. What do you think?

Reply:
@bijington it was mixed behavior before.

If a user used an event, it would never add a tap gesture.
If a user used a command, it would always add a tap gesture.

So, making it default true, would change the behavior for people using events. And, defaulting it to false changes the behavior for people using commands

Reply:
I see. OK then we can just make it clear that it is opt-in on the docs. We'll have to break someone so this is fine.

Thanks I'll merge it now